addons/account_peppol/models/res_config_settings.py

```diff
@@ -14,6 +14,7 @@ class ResConfigSettings(models.TransientModel):
         string='EDI user',
         compute='_compute_account_peppol_edi_user',
     )
+    account_peppol_edi_mode = fields.Selection(related='account_peppol_edi_user.edi_mode', readonly=True)
```

Comment:
```suggestion
    account_peppol_edi_mode = fields.Selection(related='account_peppol_edi_user.edi_mode')
```
`readonly` is true by default for related fields